Cargo.toml

```diff
@@ -112,16 +112,22 @@ pre-release-replacements = [
 [badges]
 codecov = { repository = "clap-rs/clap-verbosity-flag" }
 
+[features]
+serde = ["dep:serde", "log/serde"]
+
 [dependencies]
 log = "0.4.1"
 clap = { version = "4.0.0", default-features = false, features = ["std", "derive"] }
+serde = { version = "1.0", optional = true, features = ["derive"] }
 
 [dev-dependencies]
 clap = { version = "4.5.4", default-features = false, features = ["help", "usage"] }
 env_logger = "0.11.3"
 tracing = "0.1"
 tracing-subscriber = "0.3"
 tracing-log = "0.2"
+serde_test = "1.0.177"
+rstest = "0.22.0"
```

Comment:
I'd rather not use this at this time

Reply:
Ok, I'll remove rstest. Do you have a preference for writing tests that only vary in their parameterization to replace rstest with? E.g. internal closure, helper functions, repeated code, assertions with test specific messages, some other way?

For curiousity's sake, what's the rationale behind avoiding rstest?

Reply:
Removed in 2dda2ab